Source/CesiumRuntime/Private/CesiumEncodedFeaturesMetadata.cpp

```diff
@@ -130,48 +130,23 @@ std::optional<EncodedFeatureIdSet> encodeFeatureIdTexture(
   if (pMappedUnrealImageIt) {
     encodedFeatureIdTexture.pTexture = pMappedUnrealImageIt->Pin();
   } else {
-    encodedFeatureIdTexture.pTexture = MakeShared<LoadedTextureResult>();
-    encodedFeatureIdTexture.pTexture->sRGB = false;
-    // TODO: upgrade to new texture creation path
-    encodedFeatureIdTexture.pTexture->textureSource = LegacyTextureSource{};
+    // Copy the image, so that we can keep a copy of it in the glTF.
+    CesiumGltf::ImageCesium imageCopy(*pFeatureIdImage);
+    encodedFeatureIdTexture.pTexture =
+        MakeShared<LoadedTextureResult>(std::move(*loadTextureAnyThreadPart(
+            imageCopy,
+            TextureAddress::TA_Clamp,
+            TextureAddress::TA_Clamp,
```

Comment:
I just realized we didn't change this for https://github.com/CesiumGS/cesium-native/pull/793. But this can look exactly the same as the code in `encodePropertyTexture`, where we find the matching `TextureAddress`